templates/patterns/gallery/pattern-gallery.html.twig

```diff
@@ -12,6 +12,12 @@
 {% set items = bcl_gallery_items(items|default([])) %}
 {% set icon_path = bcl_icon_path %}
 
+{%- if title_url is not empty -%}
```

Comment:
I think its a misunderstanding, let's rather remove all these new fields to prevent opening a pandoras box of heading fields. And later if they are needed it will be a separate ticket handing them all

Reply:
done.